tests/template_lists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.14) #Required for FetchContent_MakeAvailable()
```

Comment:
Since these tests are being generated and consumed on the fly you may as well just use whatever version of CMake is being used to configure the test to avoid compatibility issues.